ctags/main/entry.c

```diff
@@ -366,8 +366,8 @@ extern void openTagFile (void)
 static int replacementTruncate (const char *const name, const long size)
 {
 	char *tempName = NULL;
-	FILE *fp = tempFile ("w", &tempName);
-	fclose (fp);
+	MIO *mio = tempFile ("w", &tempName);
+	fclose (mio);
```

Comment:
> only if it's actually found on the system

Yeah, of course, I was just commenting the fact that I didn't see any warnings from the compiler because it didn't get compiled on my system.